docroot/modules/custom/cs_generated_content/src/CsGeneratedContentAssetGenerator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\cs_generated_content;
```

Comment:
custom assets generator to point to custom asset s directory with overridden assets (images of different sizes in this case)